pom.xml

```diff
@@ -154,6 +134,12 @@
 			<version>2.12.4</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-all</artifactId>
+			<version>1.3</version>
```

Comment:
avoid to use fixed version here, define a property instead as for other dependencies.